### ISSUE TITLE
word2vec.c file is missing

### ISSUE THREAD

**Issue by alexanderpanchenko** [azpoliak/skip-gram-tensor#1]
to reproduce: 

```
~/skip-gram-tensor/hyperref-tensor$ make word2vec
make: *** No rule to make target 'word2vec.c', needed by 'word2vec'.  Stop.
```

adding the word2vec.c from https://github.com/dav/word2vec/ solves the issue

**Comment by azpoliak** [comment]
Thanks for letting us know! I just added the original `word2vec.c` in commit ac813fb3b1b4793cfeae580b0be1e65a0c2025e0 to solve the issue.

**Comment by alexanderpanchenko** [comment]
also note that the ```chmod +x *.sh``` in the makefile causes an error (there is no sh files)

**Comment by azpoliak** [comment]
just fixed it in 484e6e461913172d97855f036238f7fba3b73fa7 and ran `make` without any issues. thanks!

**Comment by alexanderpanchenko** [comment]
thanks!
